src/index.html

```diff
@@ -3,11 +3,86 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Product cards</title>
+          content="width=device-width,
+                  user-scalable=no,
+                  initial-scale=1.0,
+                  maximum-scale=1.0,
+                  minimum-scale=1.0"
+    >
+    <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
```

Comment:
it's ok according to the checklist, but further you can do not leave empty lines if the tag contains only 1 line